### PR TITLE
added translations for Croatian, Slovenian, Serbian, and Macedonian

### PR DESCRIPTION
**Pull request by alenNBH**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

# Type of change.

- [ ] FIX - a PR of this type patches a bug.
- [ ] FEATURE - a PR of this type introduces a new feature.
- [ ] BUILD - a PR of this type introduces build changes.
- [ ] CI - a PR of this type introduces CI changes.
- [ ] DOCS - a PR of this type introduces DOCS improvement.
- [ ] STYLE - a PR of this type introduces style changes.
- [ ] REFACTOR - a PR of this type introduces refactoring.
- [ ] PERFORMANCE - a PR of this type introduces performance changes.
- [ ] TEST - a PR of this type adds more tests.
- [X] CHORE - a PR introduces other changes than the specified above.

### CODE REVIEW

File: packages/core/src/lib/locale/sl.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      "getAWallet": "Ustvarite denarnico",
+      "useAWallet": "Uporabite denarnico, da bi zavarovali in upravljali s svoja NEAR digitalna sredstva, in se prijavite v katero koli aplikacijo ekosistema NEAR",
+      "connectionFailed": "Povezava ni bila uspešna.",
+      "connectionSuccessful": "Povezava je bila uspešna.",
+      "connected": "Vaša denarnica je povezana.",
+      "connectingTo": "Povezovanje z",
+      "connectingMessage": {
+        "injected": "Potrdite povezavo v oknu razširitve",
+        "browser": "Po preusmeritvi potrdite povezavo v denarnici",
+        "hardware": "Potrdite povezavo s hladno denarnico"
```
Comment (DamirSQA) [inline packages/core/src/lib/locale/sl.json:19]: "bridge": "Confirm the connection in the wallet" is missing

File: packages/core/src/lib/locale/mk.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      "youllNeedToInstall": "Треба да инсталирате",
+      "toContinueAfterInstalling": "за да продолжите. По инсталирањето",
+      "refreshThePage": "Освежете ја страната.",
+      "open": "Отворете"
+    },
+    "qr": {
+      "copiedToClipboard": "Копирано на клипбордот",
+      "failedToCopy": "Неуспешно копирање на клипборд",
+      "scanWithYourMobile": "Скенирајте со вашиот телефонски уред",
+      "copyToClipboard": " Копирајте на клипборд",
```
Comment (DamirSQA) [inline packages/core/src/lib/locale/mk.json:53]: Remove space in front of K

**Review by DamirSQA — CHANGES_REQUESTED** [review]

**Comment by alenNBH** [comment]
Hi, I have added translations for the Export accounts sections and have resolved open changes. 
Let me know if there is anything left that needs to be fixed/resolved.

File: packages/core/src/lib/locale/sr.json [hunk 3]
```diff
@@ -27,32 +28,71 @@
       "enterYourPreferredHDPath": "Унесите жељену путању за ХД, а затим скенирајте све активне налоге.",
```
Comment (DamirSQA) [inline packages/core/src/lib/locale/sr.json:28]: "enterYourPreferredHDPath": "Унесите жељену путању за ХД, а затим скенирајте све активне налоге." => 
"enterYourPreferredHDPath": "Унесите жељену жељену ХД путању, а затим скенирајте све активне налоге."

File: packages/core/src/lib/locale/hr.json [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+        "deselectAll": "Makni odabir sa svih",
+        "selectAll": "Odaberi sve",
+        "unavailable": "Prijenos nije dostupan",
+        "error": "Račun ne postoji",
+        "warningLedger": "Potrebna ledger podrška",
+        "noBalance": "Račun nema sredstava"
+      },
+      "getPassphrase": {
+        "title": "Kopiraj privremenu lozinku",
+        "desc": "Bit će potrebno unijeti ovu lozinku na početku izvoza računa na drugi wallet.",
```
Comment (DamirSQA) [inline packages/core/src/lib/locale/hr.json:85]: "desc": "Bit će potrebno unijeti ovu lozinku na početku izvoza računa na drugi wallet." => "desc": "Bit će potrebno unijeti ovu lozinku na početku izvoza naloga na drugi wallet."

File: packages/core/src/lib/locale/sr.json [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+      },
+      "selectAccounts": {
+        "title": "Изаберите налоге за пренос.",
+        "button": "Добијте приступну фразу",
+        "deselectAll": "Поништите избор",
+        "selectAll": "Изаберите све",
+        "unavailable": "Трансфер није доступан",
+        "error": "Рачун не постоји",
+        "warningLedger": "Потребна подршка за Ledger",
+        "noBalance": "Рачун није финансиран"
```
Comment (DamirSQA) [inline packages/core/src/lib/locale/sr.json:81]: "noBalance": "Рачун није финансиран" => "noBalance": "Налог није финансиран"

File: packages/core/src/lib/locale/hr.json [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+      },
+      "selectAccounts": {
+        "title": "Odaberi račune za prijenos",
+        "button": "Generiraj lozinku",
+        "deselectAll": "Makni odabir sa svih",
+        "selectAll": "Odaberi sve",
+        "unavailable": "Prijenos nije dostupan",
+        "error": "Račun ne postoji",
+        "warningLedger": "Potrebna ledger podrška",
+        "noBalance": "Račun nema sredstava"
```
Comment (DamirSQA) [inline packages/core/src/lib/locale/hr.json:81]: "noBalance": "Račun nema sredstava" => "noBalance": "Nalog nema sredstava"

**Review by DamirSQA — CHANGES_REQUESTED** [review]

**Comment by DamirSQA** [comment]
> Hi, I have added translations for the Export accounts sections and have resolved open changes. Let me know if there is anything left that needs to be fixed/resolved.

Hi @alenNBH, and adding `"exportAccounts"`. and addressing required changes. I've added some comments, so if would be great if they could be addressed as well.

**Review by DamirSQA — APPROVED** [review]

**Pull request merged**